rts/Sim/MoveTypes/GroundMoveType.cpp

```diff
@@ -2993,7 +2939,8 @@ void CGroundMoveType::HandleFeatureCollisions(
 		const float collideeMassScale = std::clamp(1.0f - r2, 0.01f, 0.99f);
 
 		forceFromMovingCollidees += colResponseVec * colliderMassScale;
-		moveFeatures.push_back(std::make_tuple(collidee, -colResponseVec * collideeMassScale));
+
+		comp.moveFeatures[curThread].emplace_back(FeatureMoveEvent(collider->id, collider, collidee, -colResponseVec * collideeMassScale));
```

Comment:
I think `emplace` takes constructor args so you don't need to repeat the type name and construct yet another copy? Ditto in events above
```suggestion
		comp.moveFeatures[curThread].emplace_back(collider->id, collider, collidee, -colResponseVec * collideeMassScale);
```

Reply:
Yes, it can. Thanks for reminding me!